src/js/views/patients/worklist/filters/date-filter_state.js

```diff
@@ -22,4 +23,54 @@ export default Backbone.Model.extend({
       selectedMonth: null,
     });
   },
+  incrementBackward() {
+    if (this.get('selectedDate')) {
+      this.setDate(dayjs(this.get('selectedDate')).subtract(1, 'day'));
```

Comment:
I think this will be useful enough to keep, mainly for the null/undefined check. I think there are at least a few instances where we can simplify some logic with that.

Reply:
ah.. yeah good point.  well might as well use it then?